pkgs/applications/video/animdl/default.nix

```
@@ -0,0 +1,68 @@
+{ lib,
+  python3Packages,
+  buildPythonApplication,
+  fetchFromGitHub,
+  poetry-core,
+  anchor-kr,
+  anitopy,
+  click,
+  cssselect,
+  httpx,
+  lxml,
+  packaging,
+  pkginfo,
+  pycryptodomex,
+  pyyaml,
+  regex,
+  rich,
+  tqdm,
+  yarl
+}:
+buildPythonApplication rec {
+  pname = "animdl";
+  version = "1.7.17";
+
+  src = fetchFromGitHub {
+    owner = "justfoolingaround";
+    repo = "${pname}";
+    # Using the commit hash because upstream does not have releases. https://github.com/justfoolingaround/animdl/issues/277
+    rev = "fd3277fb919c4b5e7ef42daba0bd001695a9c124";
+    hash = "sha256-pgtrStorkIJJ3oIGgRjNWYwcQ5jUC8IS5iI9CyDJkzE=";
+  };
+
+  format = "pyproject";
+  nativeBuildInputs = [
+    poetry-core
+    cssselect
+  ];
+  propagatedBuildInputs = [
+    anchor-kr
+    anitopy
+    click
+    httpx
+    lxml
+    packaging
+    pkginfo pyyaml
+    pycryptodomex
+    regex
+    rich
+    tqdm
+    yarl
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'pycryptodomex = "~=3.14.1"' 'pycryptodomex = "*"' \
+      --replace 'rich = ">=13.3.1,<13.3.4"' 'rich = "*"' \
+      --replace 'version = "4.9.1"' 'version = "*"' \
+      --replace 'comtypes = "~=1.1.11"' ' '
+  '';
+
+  meta = with lib; {
+    description = "A highly efficient, powerful and fast anime scraper";
+    homepage = "https://github.com/justfoolingaround/animdl";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ PassiveLemon ];
```

```suggestion
    maintainers = with maintainers; [ passivelemon ];
```
This has to be lowercase because it's defined in lower case in `maintainers/maintainer-list.nix` and afaik we only allow lower case maintainer entrys. Please change this for every package, after that it should be good to go. You can check if the package build fines localy if you execute `nix-build -A $pname` or python3Package.$pname etc.. in the root of your nixpkgs repo

---

Done.